packages/kotti-comment/src/Comment.vue

```diff
@@ -9,7 +9,7 @@
 			<div
 				v-if="!isInlineEdit"
 				class="comment__message"
-				v-text="inlineMessage"
+				v-html="inlineMessage"
```

Comment:
This means that Kotti, by default, would be cross-site-scripting vulnerable. I don’t think that’s a good idea since it will eventually lead to someone having a **critical** security vulnerability.

For example, if I wouldn’t have seen this PR and upgraded Kotti after this is merged, any user in AMPI could comment `<script>alert('pwned')</script>` and execute arbitrary javascript in the browser of anyone seeing their comment.

Quoting from The Vue documentation:

> Dynamically rendering arbitrary HTML on your website can be very dangerous because it can easily lead to XSS vulnerabilities. Only use HTML interpolation on trusted content and **never on user-provided content.**
> — https://vuejs.org/v2/guide/syntax.html#Raw-HTML

That’s also why I **really** prefer [React’s equivalent of `v-html`](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml):

```jsx
function createMarkup() {
  return {__html: 'First &middot; Second'};
}

function MyComponent() {
  return <div dangerouslySetInnerHTML={createMarkup()} />;
}
```

If you **absolutely** need this feature, you should make it opt-in and use a similarly "scary" name as React, for example by adding a `dangerouslyEnableHTML` property to `<KtComment/>`.

Reply:
@FlorianWendelborn  Correct me if I am wrong, since vue.js 2, the `v-html` already blocked dangerous scripts and only allow few html tags.

Reply:
@FlorianWendelborn how's now

Reply:
@ostoc It’s possible that `vue` attempted to block parts of the issue, but `<img src="INSERT-SOME-IMAGE-HERE" onload="alert('arbitrary-javascript')">` works.